### PR TITLE
New/test

### PR DESCRIPTION
**Pull request by ijas-ws**
### Ticket Link
---------------------------------------------------

### Related Links
---------------------------------------------------

### Description
---------------------------------------------------

### Steps to Reproduce / Test
---------------------------------------------------

### Request
---------------------------------------------------

### Response
---------------------------------------------------

### CODE REVIEW

File: pkg/api/api.go [hunk 1]
```diff
@@ -41,50 +41,55 @@ func Start(cfg *config.Configuration) (*echo.Echo, error) {
...
-		return nil, err
-	}
+	// jwt, err := jwt.New(
+	// 	cfg.JWT.SigningAlgorithm,
+	// 	os.Getenv("JWT_SECRET"),
+	// 	cfg.JWT.DurationMinutes,
+	// 	cfg.JWT.MinSecretLength)
+	// if err != nil {
+	// 	return nil, err
+	// }
```
Comment (rameez-wed) [inline pkg/api/api.go]: remove if not required

File: pkg/api/api.go [hunk 2]
```diff
@@ -41,50 +41,55 @@ func Start(cfg *config.Configuration) (*echo.Echo, error) {
...
 	}
-
+	
 	// graphql apis
-	graphqlHandler.AroundOperations(func(ctx context.Context, next graphql2.OperationHandler) graphql2.ResponseHandler {
-		return authMw.GraphQLMiddleware(ctx, jwt, next)
-	})
+	// graphqlHandler.AroundOperations(func(ctx context.Context, next graphql2.OperationHandler) graphql2.ResponseHandler {
+	// 	return authMw.GraphQLMiddleware(ctx, jwt, next)
+	// })
```
Comment (rameez-wed) [inline pkg/api/api.go]: remove if not required

File: resolver/article_mutations.resolvers_test.go [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+               req: gqlmodels.ArticleCreateInput{
+                    Title: testutls.MockArticle().Title.String,
+                    AuthorID: strconv.Itoa(testutls.MockArticle().AuthorID.Int),
+               },
+               wantResp: &gqlmodels.Article{
+                    ID: fmt.Sprint(testutls.MockArticle().ID),
+                    Title: convert.NullDotStringToPointerString(testutls.MockArticle().Title),
+                    // CreatedAt: convert.NullDotTimeToPointerInt(testutls.MockArticle().CreatedAt),
+                    // UpdatedAt: convert.NullDotTimeToPointerInt(testutls.MockArticle().UpdatedAt),
+                    // DeletedAt: convert.NullDotTimeToPointerInt(testutls.MockArticle().DeletedAt),
```
Comment (rameez-wed) [inline resolver/article_mutations.resolvers_test.go]: remove if not required

File: pkg/api/api.go (hunk 2 above)
Comment (ijas-ws) [reply pkg/api/api.go]: Ok

File: resolver/article_mutations.resolvers_test.go (hunk 3 above)
Comment (ijas-ws) [reply resolver/article_mutations.resolvers_test.go]: Ok

File: pkg/api/api.go (hunk 1 above)
Comment (ijas-ws) [reply pkg/api/api.go]: Ok

**Review by rameez-wed — APPROVED** [review]

**Pull request merged**